common/src/main/resources/common/delete-resource.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Delete resource
```

Comment:
Can add @ignore indicator to exclude this reusable feature from a list of testable features